var/spack/repos/builtin/packages/rocksdb/package.py

```diff
@@ -93,6 +94,9 @@ def install(self, spec, prefix):
         env["CFLAGS"] = " ".join(cflags)
         env["PLATFORM_FLAGS"] = " ".join(ldflags)
 
+        if "+werror" not in self.spec:
```

Comment:
Suggest removing the double negative:

```suggestion
        if "~werror" in self.spec:
```